telegram/ext/_conversationhandler.py

```diff
@@ -564,6 +581,7 @@ async def _initialize_persistence(
         return self._conversations
 
     def _get_key(self, update: Update) -> Tuple[int, ...]:
+        """Gets the chat/user/(inline)message id of the chat this update is associated with."""
```

Comment:
not quite. the conversation key is a combination of those. maybe just "Build the conversation key associated with the update"?